test/parallel/xyk-pallet.API.pool.deactivatedPool.test.ts

```diff
@@ -0,0 +1,316 @@
+/*
+ *
+ * @group poolliquidity
+ */
+import { jest } from "@jest/globals";
+import { Keyring } from "@polkadot/api";
+import { getApi, initApi, mangata } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { User } from "../../utils/User";
+import { getEnvironmentRequiredVars, stringToBN } from "../../utils/utils";
+import { Xyk } from "../../utils/xyk";
+import { BN } from "@polkadot/util";
+import "jest-extended";
+import {
+  activateLiquidity,
+  claimRewards,
+  compoundRewards,
+  createPool,
+  getLiquidityAssetId,
+  getRewardsInfo,
+  mintLiquidity,
+  promotePool,
+  sellAsset,
+} from "../../utils/tx";
+import {
+  getBalanceOfPool,
+  getEventResultFromMangataTx,
+} from "../../utils/txHandler";
+import {
+  ExtrinsicResult,
+  waitForRewards,
+  waitSudoOperationFail,
+} from "../../utils/eventListeners";
+import { BN_ZERO } from "@mangata-finance/sdk";
+import {
+  checkLastBootstrapFinalized,
+  scheduleBootstrap,
+  setupBootstrapTokensBalance,
+} from "../../utils/Bootstrap";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(2500000);
+
+process.env.NODE_ENV = "test";
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+let testUser1: User;
+let testUser2: User;
+let sudo: User;
+let keyring: Keyring;
+let token1: BN;
+let liquidityId: BN;
+const defaultCurrencyValue = new BN(250000);
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+  keyring = new Keyring({ type: "sr25519" });
+
+  // setup users
+  sudo = new User(keyring, sudoUserName);
+
+  await setupApi();
+
+  [testUser1, testUser2] = setupUsers();
```

Comment:
plz, do not reuse the users unless is strictly necessary.

Reply:
Ok, changed